src/GUI/.editorconfig

```diff
@@ -0,0 +1,137 @@
+# Rules in this file were initially inferred by Visual Studio IntelliCode from the C:\Work\SMProjects\contributions\EFCorePowerTools\src\GUI codebase based on best match to current usage at 18/11/2020
```

Comment:
Why is this file included?

Reply:
My fault. I follow 3 projects (for my customers) and this one. Each project uses different code style. I use this file to adapt to each project style but i shouldn't have committed it. I'll remove it.